hypothesis-python/tests/cover/test_functions.py

```diff
@@ -125,3 +129,61 @@ def test_functions_strategy_with_kwonly_args(f):
         f(1, 2)
     f(1, kwonly_arg=2)
     f(kwonly_arg=2, arg=1)
+
+
+def pure_func(arg1, arg2):
+    ...
```

Comment:
Style nit: I'd prefer `pass` here, to indicate that the body is intentionally empty - `...` often implies that a non-empty body has been ommitted.

(I can't find the reference now, but this semantic difference was used to justify keeping the `pass` statement in a design discussion some years ago)